### ISSUE TITLE
An option to disable registration in self-hosted setup

### ISSUE THREAD

**Issue by thelazyoxymoron** [solidtime-io/solidtime#196]
Would love to have an option to disable user registrations

**Comment by thelazyoxymoron** [comment]
Environment variable should be sufficient in this case

**Comment by micalm** [comment]
I agree that an env would be enough; this is how [Gitea](https://docs.gitea.com/administration/config-cheat-sheet#service-service) does it.

They do have an registration (or rather user creation) view for the admin, which would require additional consideration for solidtime:
1. Do users get sent a password via email or do they have to set it themselves?
2. Are personal organizations created or are users forced to live in a certain organization?
3. Something else I'm probably missing now

---

Another approach that I think would be similar in effort required is a Slack-like `ALLOWED_EMAILS` or something, where guests **can** register, as long as they have an email in a domain allowed by the instance admin. This doesn't require any changes or new flows except a small message that `you can't register from that email address, bye`.

**Comment by thelazyoxymoron** [comment]
Isn't there already a "new user" creation functionality for admins in Solidtime as well? I can see one in `/admin/users` page. Or am I missing something?
